### PR TITLE
ENH Add elemental docs

### PR DESCRIPTION
**Pull request by emteknetnz**
Issue https://github.com/silverstripe/silverstripe-elemental/issues/985

### CODE REVIEW

**Review by GuySartorelli — APPROVED** [review]
LGTM, will merge after docs are merged

File: sources-user.js [hunk 1]
```diff
@@ -18,6 +18,15 @@ module.exports = [
...
   },
+  {
+    resolve: `gatsby-source-git`,
+    options: {
+      name: `user--5--optional_features/elemental`,
+      remote: `https://github.com/silverstripe/silverstripe-elemental`,
+      branch: `5.2`,
+      patterns: `docs/en/userguide/**`,
+    }
+  },
```
Comment (GuySartorelli) [inline sources-user.js:29]: Just realised this is a duplicate of the `dnadesign/silverstripe-elemental` entry that's already in this file. Please update that entry.

Please also use the `5` branch for user docs. We so rarely update these docs that we can effectively treat the next minor branch as stable.

**Review by GuySartorelli — CHANGES_REQUESTED** [review]

File: sources-user.js (hunk 1 above)
Comment (emteknetnz) [reply sources-user.js]: Removed, just use the existing one

**Review by GuySartorelli — APPROVED** [review]
As before, will merge after docs are merged

**Pull request merged**